### ISSUE TITLE
Chapter 9 git commit comment change

### ISSUE THREAD

**Issue by lenixlenixlenix** [wsvincent/restapiswithdjango#118]
Original command in book is  `git commit -m "add schema and documentation"`.
Should be fixed to  `git commit -m "altered to viewsets and routers"`, because chapter is about ViewSets and Routers.
![image](https://github.com/wsvincent/restapiswithdjango/assets/148846450/d59d243b-f9fc-49bc-b48d-26c62aa519eb)

**Comment by wsvincent** [comment]
Yes, exactly right! I'm updating this for the next edition. Thank you for taking the time to submit an Issue on it.
